### ISSUE TITLE
DST Staging Review: Keyboard focus lost when closing the delete modal (without deleting)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#3082]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** File Input Component
**Staging Review ticket:** [Staging Review: (DS) Converted File Input Component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77552)

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**This is an issue with the:** code
**High-priority:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Testing in Storybook, keyboard focus seems to be lost when you press "No, keep this" in the delete file modal and when you press the X close button. Losing keyboard focus can be confusing, both to sighted keyboard users and screen reader users.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Upload a file.
2. Tab to Delete. Press enter to open the delete modal.
3. Focus should be on the X. Press enter to close the modal.
4. Note that there's no visible focus indicator.
5. Press enter. Nothing happens. (So no button or link currently has focus.)
6. Press shift + tab.
7. Note that focus moves back to the Delete button.
8. Press enter to open the delete modal.
9. Press tab until focus goes to "No, keep this."
10. Press enter to close the modal.
11. Note that there's no visible focus indicator. Repeat 5, 6, 7.

</details>

## Recommended action
When closing the modal without deleting, focus should go back to the Delete button.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
